packages/editor/src/components/widgets/combobox/Combobox.css

```diff
@@ -27,21 +27,31 @@
   list-style: none;
   font-size: 12px;
   text-align: start;
-  background-color: var(--input-bg);
+  background-color: var(--background);
   padding: 0;
-  margin: 0px;
+  margin: var(--size-1) 0 0 0;
   z-index: 10;
   box-shadow: var(--editor-shadow);
+  left: var(--size-2);
+  width: calc(100% - var(--size-4));
+  border-radius: var(--border-radius);
 }
 .combobox-menu-entry {
   padding: 0.5rem;
   display: flex;
   flex-direction: column;
   gap: var(--size-2);
   cursor: pointer;
+  border-bottom: var(--seperator-border);
 }
-.combobox-menu-entry:where(.hover, .selected) {
-  background: var(--editor-bg);
+.combobox-menu-entry:last-child {
+  border-bottom: none;
+}
+.combobox-menu-entry:where(.hover) {
+  background: var(--N50);
+}
+.combobox-menu-entry:where(.selected) {
+  background: var(--N100);
```

Comment:
if you only have one "where clause", the where makes no sense, you can simply write `.combobox-menu-entry.hover`

The where is cool if you need to apply the same styles to multiple different things, e.g. 
```css
.combobox-menu-entry:where(.hover, .selected) {
  background: var(--editor-bg);
}

/*instead of */
.combobox-menu-entry.hover, 
.combobox-menu-entry.selected {
  background: var(--editor-bg);
}
```